plugins/lsp/language-servers/default.nix

```diff
@@ -596,6 +596,12 @@ with lib; let
       description = "tsserver for TypeScript";
       package = pkgs.nodePackages.typescript-language-server;
     }
+    {
+      name = "typos-lsp";
+      serverName = "typos_lsp";
+      description = "Source code spell checker for Visual Studio Code and LSP clients";
+      package = pkgs.typos-lsp;
```

Comment:
```suggestion
```
Actually, you don't even need this as we use `pkgs.${name}` for `package` by default.

Reply:
Ah I thought that was the case!